.github/workflows/build-and-push-image-build.yml

```diff
@@ -58,13 +51,12 @@ jobs:
     uses: access-nri/build-ci/.github/workflows/build-and-push-image.yml@main
     with:
       container-registry: ghcr.io
-      container-name: access-nri/build-${{ matrix.package }}-${{ matrix.compiler.name }}${{ matrix.compiler.version }}-${{ github.event.inputs.spack-packages-version }}
+      container-name: access-nri/build-${{ matrix.compiler.name }}${{ matrix.compiler.version }}-${{ github.event.inputs.spack-packages-version }}
```

Comment:
At the risk of adding more back into the container name, if we're going for monolithic "per-model" containers the model name should figure somewhere I think. Probably in the name, in which case does it also have to be an argument to the workflow under `jobs:`?

The ideal should be that we can specify a different model, e.g. access-om3, and all of this would work for that monolithic build  as well.

Don't rule out the possibility that I've misunderstood how this works and this is not required.

Reply:
No, you're correct there. We'll need to specify the overall model name (access-om2, access-om3) in the build. Will update now

Reply:
Updated!